dependencies.gradle

```diff
@@ -7,6 +7,8 @@ dependencies {
     shadowImplementation("it.unimi.dsi:fastutil:8.5.15") // Apache 2.0
     shadowImplementation("org.joml:joml:1.10.8") { transitive = false } // MIT
     shadowImplementation("com.mojang:brigadier:1.0.18") // MIT
+    // 1.3.5-SNAPSHOT has some Buffer.slice() stubs that Angelica needs.  Switch to 1.3.5+ when available
+    shadowImplementation("xyz.wagyourtail.jvmdowngrader:jvmdowngrader-java-api:1.3.5-SNAPSHOT:downgraded-8") { transitive = false } // LGPL 2.1
```

Comment:
Ideally we want to pull in at least downgraded-8 and downgraded-17 (not into shadowImplementation, probably in a new configuration in build.gradle.kts) and manually use some from() calls in the jar task in gradle to assemble a multiversion shaded copy of these so that on j17+ the j17 stubs are used for efficiency, I imagine that has the most efficiency gains and 17-25 will be much less